runtime/src/enclave_rpc/session.rs

```diff
@@ -47,8 +51,12 @@ enum SessionError {
 
 /// Information about a session.
 pub struct SessionInfo {
+    /// RAK binding.
     pub rak_binding: RAKBinding,
+    /// Verified TEE quote.
     pub verified_quote: VerifiedQuote,
+    /// Identifier of the node that endorsed the TEE.
+    pub endorsed_by: Option<PublicKey>,
```

Comment:
Is this used anywhere? I'm still not sure why the host needs to endorse `CapabilityTEE`.

Reply:
It will be needed so that the SDK module can check that the ROFL runtime is running on one of the nodes that are registered in the consensus layer, maybe part of the active committee etc.